### PR TITLE
Documentation changes for Portable and Tutorial data

### PR DESCRIPTION
**Pull request by tst-hrajamoney**

### CODE REVIEW

**Comment by tst-hrajamoney** [comment]
Have to generate HTML after all reviewers are okay with the text. Do not merge until html is generated.

File: docs/geedocs/docsrc/answer/portable/portableUserGuideWinLinux.rst [hunk 1]
```diff
@@ -272,26 +273,22 @@ Portable User Guide for Windows and Linux
       .. _Connect_Using_Google_earth_Plugin:
       .. rubric:: Connect using the Google Earth Plug-in
 
-      GEE Portable comes with preconfigured HTML pages called
-      ``hello_earth.html`` and ``hello_maps.html`` that display your
-      globe or map using the Google Earth API or Google Map API. Note
-      that the ``hello_earth.html`` sample uses the deprecated Google
-      Earth Plug-in.
+      GEE Portable comes with preconfigured HTML page called
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/portable/portableUserGuideWinLinux.rst]: **a** preconfigured HTML page... that display**s** your map

File: docs/geedocs/docsrc/answer/portable/portableUserGuideWinLinux.rst [hunk 2]
```diff
@@ -272,26 +273,22 @@ Portable User Guide for Windows and Linux
       .. _Connect_Using_Google_earth_Plugin:
       .. rubric:: Connect using the Google Earth Plug-in
 
-      GEE Portable comes with preconfigured HTML pages called
-      ``hello_earth.html`` and ``hello_maps.html`` that display your
-      globe or map using the Google Earth API or Google Map API. Note
-      that the ``hello_earth.html`` sample uses the deprecated Google
-      Earth Plug-in.
+      GEE Portable comes with a preconfigured HTML page called
+      ``hello_maps.html`` that displays your map using the Google Map API.
```
Comment (tst-jcupp) [inline docs/geedocs/docsrc/answer/portable/portableUserGuideWinLinux.rst]: "Google Maps API"

File: docs/geedocs/docsrc/answer/portable/portableUserGuideWinLinux.rst [hunk 3]
```diff
@@ -253,8 +253,9 @@ Portable User Guide for Windows and Linux
       ``http://localhost:9335`` (using any
       custom port or hostname you may have configured). Click the
       |Portable folder icon| **Folder** (outlined in red), then select the
-      globe or map you want to view. You can view only one globe or
-      map at a time.
+      map you want to view. You can view only one map at a time. Globes
+      cannot be viewed from the Portable. Use Google Earth Enterprise
```
Comment (tst-jcupp) [inline docs/geedocs/docsrc/answer/portable/portableUserGuideWinLinux.rst]: "Portable globes cannot be viewed with a web browser."

**Comment by tst-swanson** [comment]
approve again

**Pull request merged**